guide/02-api-overview/release-notes.ipynb

```diff
@@ -13,6 +13,18 @@
    "source": [
```

Comment:
<p>Need to mention fixes in PR   - https://github.com/ArcGIS/geosaurus/pull/7687</p><p> </p><ol><li>Made the calendar fied and operator parameters of filter_by_calendar_range, filter_by_attribute respectively,  case insensitive for raster collections created from datastore rasters.</li><li>RasterCollection engine was not getting set properly when engine is set to "arcpy"</li></ol><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1158/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
<p>Thanks. Added those notes.</p>